src/main/fc/fc_rc.c

```diff
@@ -169,6 +169,8 @@ static void calculateSetpointRate(int axis)
 
         angleRate = applyRates(axis, rcCommandf, rcCommandfAbs);
     }
+    // Rate limit from profile (deg/sec)
+    angleRate = constrainf(angleRate, -1.0f * currentControlRateProfile->rate_limit[axis], 1.0f * currentControlRateProfile->rate_limit[axis]);
     setpointRate[axis] = constrainf(angleRate, -SETPOINT_RATE_LIMIT, SETPOINT_RATE_LIMIT); // Rate limit protection (deg/sec)
 
     DEBUG_SET(DEBUG_ANGLERATE, axis, angleRate);
```

Comment:
This is wrong now, as `setpointRate[axis]` isn't always identical to `angleRate` with your code.

Reply:
Existing code for `angleRate < SETPOINT_RATE_LIMIT` or `angleRate > SETPOINT_RATE_LIMIT` results in `setpointRate[axis] != angleRate`. I assumed that was intended behaviour.

Also, what's the intended debug behavour? That `angleRate` not be constrained so the `DEBUG_SET` shows the un-clipped value? In which case I need to change my code...